prometheus/metric_test.go

```diff
@@ -11,6 +11,47 @@ import (
 	"github.com/segmentio/stats/v4"
 )
 
+func TestUnsafeByteSliceToString(t *testing.T) {
```

Comment:
Thanks for adding a test for this!

Reply:
This is @collinvandyck 's contribution, kudos to him!

Reply:
Hehe, yeah, after submitting my review I saw the same comment from Julien :P